RachealN/Web-Practical
# chore(setup): set up the server and the database [Finishes #167958675]

What does this PR do?
setting up a server
set u[p the database

Description of Task to be completed?
-created directories
-setup and running the server
-set up the database
installed packages

How should this be manually tested?
git clone this repo: What does this PR do?
login a user/ agent

Description of Task to be completed?
-login the user/agent
-validations implemented

How should this be manually tested?
git clone this repo:https://github.com/RachealN/PropertyPro-lite-API.git
Run npm run start to start the server
Run npm run migrations for the database

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
#167958675



Screenshots (if appropriate)
